File: Swift4/Swift4/model/PartijaDvaIgraca.swift
//
//  PartijaDvaIgraca.swift
//  Swift4
//
//  Created by Tomislav Jakopec on 22/02/2020.
//  Copyright © 2020 Tomislav Jakopec. All rights reserved.
//

import Foundation
class PartijaDvaIgraca: Partija{
    
    public override var description: String {
        let rezultat = getRezultat();
        return "Partija DVA IGRAČA, igra gotova: " + String(super.isIgraGotova()) + ", " +  super.getIgraci()[0].description + ": " +
                String(rezultat.getPrvi()) +
            " | " + super.getIgraci()[1].description + ": " + String(rezultat.getDrugi());
    }
    
}

File: JS/JS/partija/partijaDvaIgraca.js
import { Partija } from './partija.js'

export class PartijaDvaIgraca extends Partija {

    constructor(id, doKolikoSeIgra, lokacija, unosi, mjesanja, igraci) {
        super(id, doKolikoSeIgra, lokacija, unosi, mjesanja, igraci)
    }

    toString() {
        const rezultat = this.getRezultat()
        const igraci = this.getIgraci()

        return `Partija DVA IGRAČA, igra gotova: ${this.isIgraGotova()}, ` +
            `${igraci[0]}: ${rezultat.getPrvi()} | ` +
            `${igraci[1]}: ${rezultat.getDrugi()}`
    }
}